tests/CMakeLists.txt

```diff
@@ -1044,8 +1054,9 @@ add_cmdline_test(manifold-stlexport    EXPERIMENTAL OPENSCAD SUFFIX stl FILES ${
 add_cmdline_test(objexport             EXPERIMENTAL OPENSCAD SUFFIX obj FILES ${EXPORT_OBJ_TEST_FILES} ARGS --render --enable=predictible-output)
 add_cmdline_test(3mfexport             EXPERIMENTAL OPENSCAD SUFFIX 3mf FILES ${EXPORT_3MF_TEST_FILES} ARGS --enable=predictible-output)
 add_cmdline_test(offexport             EXPERIMENTAL OPENSCAD SUFFIX off FILES ${EXPORT_OFFEXPORT_TEST_FILES} ARGS --enable=predictible-output --render)
-add_cmdline_test(color-offexport       EXPERIMENTAL OPENSCAD SUFFIX off FILES ${EXPORT_OFFEXPORT_TEST_FILES} ARGS --enable=manifold --enable=colors --enable=predictible-output --render)
-add_cmdline_test(color-offimport       EXPERIMENTAL OPENSCAD SUFFIX png FILES ${EXPORT_OFFIMPORT_TEST_FILES} ARGS --enable=colors)
+add_cmdline_test(color-offexport       EXPERIMENTAL OPENSCAD SUFFIX off FILES ${EXPORT_OFFEXPORT_TEST_FILES} ARGS --enable=manifold --enable=render-colors --enable=predictible-output --render)
+add_cmdline_test(color-offimport       EXPERIMENTAL OPENSCAD SUFFIX png FILES ${EXPORT_OFFIMPORT_TEST_FILES} ARGS --enable=render-colors)
+add_cmdline_test(colorpngtest          EXPERIMENTAL OPENSCAD SUFFIX png FILES ${COLOR_TEST_FILES} ARGS --enable=render-colors)
```

Comment:
I would add `--render` here, otherwise you'll be testing previews.

Reply:
ah!!